pcsx2/gui/AppCoreThread.cpp

```diff
@@ -406,30 +422,35 @@ void AppCoreThread::ApplySettings( const Pcsx2Config& src )
 	ResetCheatsCount();
 
 	//Till the end of this function, entry CRC will be 00000000
-	if (!gameCRC.Length()) {
-		if (EmuConfig.EnableWideScreenPatches || EmuConfig.EnableCheats) {
+	if (!gameCRC.Length())
+	{
+		if (EmuConfig.EnableWideScreenPatches || EmuConfig.EnableCheats)
+		{
 			Console.WriteLn(Color_Gray, "Patches: No CRC, using 00000000 instead.");
 		}
 		gameCRC = L"00000000";
 	}
 
 	// regular cheat patches
-	if (EmuConfig.EnableCheats) {
-		if (numberLoadedCheats = LoadCheats(gameCRC, GetCheatsFolder(), L"Cheats")) {
+	if (EmuConfig.EnableCheats && (numberLoadedCheats = LoadCheats(gameCRC, GetCheatsFolder(), L"Cheats")))
+	{
```

Comment:
Scope of `numberLoadedCheats` can be reduced instead.

Same applies to `numberLoadedWideScreenPatches` and `numberDbfCheatsLoaded`.